Data/ByteString/Lazy.hs

```diff
@@ -722,6 +803,23 @@ takeWhile f = takeWhile'
             n | n < S.length c -> Chunk (S.take n c) Empty
               | otherwise      -> Chunk c (takeWhile' cs)
 
+-- | Returns the longest (possibly empty) suffix of elements
+-- satisfying the predicate.
+--
+-- @'takeWhileEnd' p@ is equivalent to @'reverse' . 'takeWhile' p . 'reverse'@.
```

Comment:
An example would be nice to have here.

Reply:
I've used `Chunk (pack [1,2]) (Chunk (pack [3,4,6])) Empty` as an example of a lazy bytestring. Hope it's not too verbose, but textual representation of a bytestring is always tricky.

Reply:
I don't think that would be helpful for users who will mostly not be aware of `ByteString`'s internal constructors. If you want to represent the bytes as numbers, you can use the `OverloadedLists` syntax, e.g. `[1,2,3,4,6]`.